dolphin/db/sqlalchemy/models.py

```diff
@@ -48,6 +48,8 @@ class AccessInfo(BASE, DolphinBase):
     """Represent access info required for storage accessing."""
     __tablename__ = "access_info"
     storage_id = Column(String(36), primary_key=True)
+    vendor = Column(String(255))
```

Comment:
In one of the other PR, we discussed to move this to Storage table, right? So we decided to move it back? Purpose?